MineSweeper.py

```diff
@@ -0,0 +1,256 @@
+import Tkinter as tk
+import random
+
+import sys
+sys.path.append('~/Library/Python/2.7/site-packages')
+
+from drawingpanel import *
+
+BOARD_HEIGHT = 300
+BOARD_WIDTH = 300
+BOX_SIZE = 20
+
+NUMBER_MINES = ((BOARD_WIDTH / BOX_SIZE - 1)**2)//10 
```

Comment:
My first thought would be that it just truncates it? So if you have 20.0 for example calling int(20.0) would return 20 or int(20.7) == 20.